lib/child-writer/index.js

```diff
@@ -170,10 +172,35 @@ exports.write = (image, drive, options) => {
     }
   }
 
+  // Timeout function that only runs if we exceed the specified timeout,
+  // terminating the server.
+  let terminationTimeout = null
+
+  /**
+   * @summary Reset the timeout countdown
+   * @function
+   * @private
+   *
+   * @example
+   * resetTerminationTimeout()
+   */
+  const resetTerminationTimeout = () => {
+    clearTimeout(terminationTimeout)
+    terminationTimeout = setTimeout(() => {
+      emitter.emit('timeout', {
+        TIMEOUT_TIME
+      })
```

Comment:
Why is `TIMEOUT_TIME` wrapped in an object here? And why is the timeout value emitted through the event?

Reply:
I assume it's good data to have should we change the value in the future, the object is because I was wondering if there's more we could possibly need, if not we can just send the value by itself.

Reply:
Yeah, I think its fine. We can use it to display a nice modal to the user saying something like "We stopped the flashing process because we didn't receive any updates in X seconds".

My only objection is that right now the property name is `TIMEOUT_TIME`, so maybe something like:

```js
{ timeout: TIMEOUT_TIME }
```

will read better?